homeassistant/components/switch/flux.py

```diff
@@ -45,7 +45,16 @@
 DEPENDENCIES = ['light']
 
 
-PLATFORM_SCHEMA = vol.Schema({
+def is_valid_transition(config):
+    """Validate transition compared to interval."""
+    transition = config[ATTR_TRANSITION]
+    interval = config[CONF_INTERVAL]
+    if transition > interval:
+        config[ATTR_TRANSITION] = interval
+    return config
+
+
+PLATFORM_SCHEMA = vol.Schema(vol.All({
```

Comment:
We should probably extend `PLATFORM_SCHEMA` from base switch component.

Reply:
This is a bugfix. We should not add arbitrary cleanups.

Reply:
Yes, but we are changing this line. Then it's better to change the validation in a separate PR. We shouldn't change lines and persist wrong code.

Reply:
Okay, I reverted that change once more.